internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -207,6 +211,9 @@ func newCollection(db *dyn.DynamoDB, tableName, partitionKey, sortKey string, op
 	if opts == nil {
 		opts = &Options{}
 	}
+	if opts.RevisionField == "" {
+		opts.RevisionField = docstore.RevisionField
```

Comment:
Maybe rename the `docstore.RevisionField` to `docstore.DefaultRevisionField`?

Reply:
Good idea. Done.